### PR TITLE
Login html

### PR DESCRIPTION
**Pull request by ChristieGriffiths**
- added a heading in login page
- added email validation tag
- removed redundant 'role' tag in login form
- added 20px margin to css file (affects whole app)
- changed console.log message for successful/unsuccessful login

### CODE REVIEW

File: package-lock.json
Comment (wimdavies) [inline package-lock.json:1]: Do you know why you have a new package-lock.json in the parent directory? Did one of you accidentally run `npm install` while in /acebook-airbenders ?

**Review by wimdavies — CHANGES_REQUESTED** [review]
Please see comment on package-lock.json

**Review by wimdavies — APPROVED** [review]
Nice one!

**Review by sarahc-dev — COMMENTED** [review]
move h1 tag to outside form element (add empty <> </> around whole component)
Add labels to form

**Pull request closed without merging**